### PR TITLE
Danielsonnenberg/point bytea tsvector

### PR DESCRIPTION
**Pull request by nezbite**

### CODE REVIEW

File: IHP/IDE/SchemaDesigner/View/Columns/Edit.hs [hunk 1]
```diff
@@ -113,6 +113,9 @@ typeSelector postgresType enumNames = [hsx|
             {option selected "Time" "Time"}
             {option selected "SERIAL" "Serial"}
             {option selected "BIGSERIAL" "Bigserial"}
+            {option selected "POINT" "Point"}
```
Comment (mpscholten) [inline IHP/IDE/SchemaDesigner/View/Columns/Edit.hs:116]: While testing I got this error in the generated haskell code:

Not in scope: type constructor or class ‘Point’

**Comment by fegu** [comment]
Is still PR stalled? I will be needing both point and tsvector in two different projects soon. Should we fix this PR or make a new one?

**Comment by mpscholten** [comment]
I think we need to update this, then we could merge

**Comment by mpscholten** [comment]
Point has been merged to master via https://github.com/digitallyinduced/ihp/commit/958d553c028b583d7a057dc66f4ee305569ee621

**Pull request closed without merging**